libraries/package-extractor/src/Utils.ts

```diff
@@ -54,3 +56,48 @@ export function remapPathForExtractorMetadata(folderPath: string, filePath: stri
   }
   return Path.convertToSlashes(relativePath);
 }
+
+/**
+ * Creates the .bin files for the extracted projects and returns the paths to the created .bin files.
+ *
+ * @param terminal - The terminal to write to
+ * @param extractedProjectFolderPaths - The paths to the extracted projects
+ */
+export async function makeBinLinksAsync(
+  terminal: ITerminal,
+  extractedProjectFolderPaths: string[]
+): Promise<string[]> {
+  const binFilePaths: string[] = [];
+  await Async.forEachAsync(
+    extractedProjectFolderPaths,
+    async (extractedProjectFolderPath: string) => {
+      const extractedProjectNodeModulesFolderPath: string = path.join(
+        extractedProjectFolderPath,
+        'node_modules'
+      );
+      const extractedProjectBinFolderPath: string = path.join(extractedProjectNodeModulesFolderPath, '.bin');
```

Comment:
```suggestion
      const extractedProjectNodeModulesFolderPath: string = `${extractedProjectFolderPath}/node_modules`;
      const extractedProjectBinFolderPath: string = `${extractedProjectNodeModulesFolderPath}/.bin`;
```

Just make sure that works on Windows.

Reply:
Updated code, will validate Windows support

Reply:
Validated